scripts/generate_result.sh

```diff
@@ -49,20 +52,22 @@ do
   # Check if challenge MQL file exists
   mqlFile="$CHALLENGES_DIR/$chName".js
   if [ ! -f $mqlFile ]; then
-    updateEvaluation "$chName" "$chDesc" 1
+    updateEvaluation "$chName" "$chDesc" $INCORRECT_GRADE
     continue
   fi
-  # Exec query into mongo container
-  mql=$(cat "$mqlFile")
-  scripts/exec.sh "$mql" &> "$resultPath"
+  # Exec query into mongo container and sort result
+  mql=$(cat "$mqlFile" | sed -r "s/;+$//")
+  scripts/exec.sh "$mql" | jq -S &> "$resultPath"
```

Comment:
```suggestion
  scripts/exec.sh "$mql" | jq ._batch[] -S &> "$resultPath"
```
Faço essa sugestão na tentativa de manter os arquivos do `expected-results` mais limpo. A forma que está, aparece algumas informações desnecessárias,

Reply:
@cleytonoliveira consegue exemplificar o que mudaria nesta situação ou passar um projeto que seria afetado por isso? Só pra eu poder validar aqui se o código ta rodando ok antes do merge

Reply:
Na forma que esta hoje, por exemplo o arquivo `desafio5` da pasta `.trybe/expected-results` deveria estar da seguinte forma:

```
{
  "_batch": [
    {
      "title": "The Heat"
    }
  ],
  "_collName": "movies",
  "_cursorHandle": {},
  "_cursorid": {
    "$numberLong": "0"
  },
  "_db": {
    "_mongo": {
      "_defaultSession": {
        "_isExplicit": false,
        "_serverSession": {
          "handle": {}
        }
      },
      "_readMode": "commands",
      "defaultDB": "aggregations",
      "host": "127.0.0.1:27017",
      "slaveOk": false
    },
    "_name": "aggregations",
    "_session": {}
  },
  "_ns": "aggregations.movies",
  "_useReadCommands": true
}
```

O que estou sugerindo é para que fique assim:

```
{
  "title": "The Heat"
}
```